fcn_conf.py

```diff
@@ -26,6 +28,7 @@ def register_node_now(op_code, class_reference):
             op_code, FC_NODES[op_code]
         ))
     FC_NODES[op_code] = class_reference
+    OP_NODE_FREE_ID = max(OP_NODE_FREE_ID, op_code + 1)
```

Comment:
I had to put `global` in front of the variable `OP_NODE_FREE_ID`, otherwise you get the error message
_**UnboundLocalError: local variable 'OP_NODE_FREE_ID' referenced before assignment**_. But nevertheless a great solution, thank you!